### ISSUE TITLE
Get hosted content Stream in the latest version of Graph API (5.3.0)

### ISSUE THREAD

**Issue by shirkhanyanhripsime** [microsoftgraph/msgraph-sdk-dotnet#1789]
I am using Graph SDK. After upgrading Graph Api to the latest version (5.3.0), I can no longer use the following request via SDK:
var request = _client
				.Chats[chatId]
				.Messages[messageId]
				.HostedContents[hostedContentId]
				.Content.Request();
Content and Request are not available.
![image](https://user-images.githubusercontent.com/129367500/228756858-4311e346-4bb9-466e-82d7-8c553304663a.png)
According to the documentation, the snippet for retrieving hosted content bytes is not available for c#. https://learn.microsoft.com/en-us/graph/api/chatmessagehostedcontent-get?view=graph-rest-1.0&tabs=csharp
So basically, I want to access the following endpoint `https://graph.microsoft.com/v1.0/chats/{chat-id}/messages/{message-id}/hostedContents/{hosted-content-id}/$value`.

What can be used instead? How can I download the hosted contents?

**Comment by andrueastman** [comment]
Related to https://github.com/microsoft/OpenAPI.NET.OData/issues/314 and should be resolved on the next metadata update. 

In the meantime, the request can be made as 

```cs
        var requestInformation = graphClient.Chats["{chat-id}"].Messages["{chatMessage-id}"].HostedContents["{chatMessageHostedContent-id}"].ToGetRequestInformation();
        requestInformation.URI = new Uri(requestInformation.URI.OriginalString + "/$value");
        var streamResponse = await graphClient.RequestAdapter.SendPrimitiveAsync<Stream>(requestInformation);
```
